ERROR_DETAILS = {
    401: 'Could not validate the credentials',
    409: 'Username already exists',
    500: 'Internal Server Error',
    404: 'Requested resource could not be found',
    'wallet_empty': 'User wallet is empty!',
    'low_user_balance': 'User wallet balance is low for the transaction',
    'invalid_amount': 'Invalid amount entered! Please enter positive amount',
    'self_transfer': 'Cannot transfer to the same account wallet!',
    'receiver_not_found': 'Receiver does not exist',
    'invalid_credentials_format': 'Invalid username or password format',
    'invalid_int_input': 'non numeric value entered ',
    'no_records': 'No records found!',
    'value_overflow': 'Requested value is too large!',
    'invalid_month_year': 'Invalid month and year entered!'
}
